app/views/posts/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>This is a lists of all Posts written by our Unique User John Doe😎</h1>
```

Comment:
- [ ] [optional] @over-geek It would have been nice if you used placeholder texts that relate to what the page is displaying eg in this page it would be `Here is a list of all users`, this applies to all the pages.